package.json

```diff
@@ -16,8 +17,13 @@
   "bugs": {
     "url": "https://github.com/ashtuchkin/u2f/issues"
   },
+  "scripts": {
+    "build": "./node_modules/.bin/tsc -p ."
+  },
   "homepage": "https://github.com/ashtuchkin/u2f",
   "devDependencies": {
-    "mocha": "2"
+    "@types/node": "^7.0.22",
+    "mocha": "2",
```

Comment:
thank you!

Reply:
removed @types/node and added ts-node instead.